app/models/helpers/health_check_types.rb

```diff
@@ -4,5 +4,14 @@ class HealthCheckTypes
     PROCESS = 'process'.freeze
     HTTP = 'http'.freeze
     NONE = 'none'.freeze
+
+    def self.constants_to_array
+      [
+        HTTP,
+        NONE,
```

Comment:
I don't understand why we have `NONE` at all, do you?  Trying to set a health check to none via the API results in the following error:

```
± sb |readiness-health-checks-story-2-in-db → origin {3} ?:1 ✗| → cf curl -X PATCH v3/processes/c6493e8f-84b0-4e84-be1f-7156b7704039 -d '{"health_check": {"type": "none","data":  {  }}}'
{
   "errors": [
      {
         "detail": "Health check type must be \"port\", \"process\", or \"http\"",
         "title": "CF-UnprocessableEntity",
         "code": 10008
      }
   ]
}
```

Reply:
It looks like it used to be an option, but no longer is.[ It looks like for awhile there was backwards compatible logic where it would map none -> process](https://github.com/cloudfoundry/cloud_controller_ng/blob/f47b8ebaaa81904d8e03ff92b978afa73815561c/app/messages/app_manifest_message.rb#L41). I suspect it is dead code now and could be ripped out. If you want we can do it in another story.

Reply:
It'd be nice to clean up, but `NONE`'s presence doesn't seem too onerous right now.  I'll leave it to you to decide if you want to do it, and if not I'll create a story/github issue to track it.

Reply:
I added a story in our epic to do it.